models/users-model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection");
+
+const checkUserExists = async ({ username }) => {
```

Comment:
PSQL should throw an error if the user doesnt exist, so I'm not sure this is needed🤔 although nicely done!